stories/demos/CommonFeatures.tsx

```diff
@@ -223,7 +224,7 @@ function createRows(): readonly Row[] {
 
 export function CommonFeatures() {
   const [rows, setRows] = useState(createRows);
-  const [[sortColumn, sortDirection], setSort] = useState<[string, SortDirection]>(['id', 'NONE']);
+  const [sortColumns, setSortColumns] = useState<SortColumn[]>([]);
```

Comment:
I think we can create a separate example. In the future we can group examples like
- Column
  - ColSpan
  - Multisort
  - Reorder
- Row
  - Group
  - Reorder
  - NoRows
etc